source/fundamentals/monitoring.txt

```diff
@@ -398,3 +398,31 @@ Oracle:
 For more information on the ``JMXConnectionPoolListener`` class, see
 the API Documentation for
 `JMXConnectionPoolListener <{+api+}/apidocs/mongodb-driver-core/com/mongodb/management/JMXConnectionPoolListener.html>`__.
+
+Include the Driver in Your Distributed Tracing System
+-----------------------------------------------------
+
+If you use a **distributed tracing system**, you can include event data from the
+driver in your system. A distributed tracing system is an application that
+tracks requests as they propagate throughout different services in a
+service-oriented architecture.
+
+If you use the driver in a `Spring Cloud <https://spring.io/projects/spring-cloud>`__
+application, use
+`Spring Cloud Sleuth <https://spring.io/projects/spring-cloud-sleuth>`__ to
+include MongoDB event data in the
+`Zipkin <https://zipkin.io/>`__ distributed tracing system.
+
+If you do not use Spring Cloud or need to include driver event data in a distributed
+tracing system other than Zipkin, you must write a command event listener that
+manages spans for your desired distributed tracing system. To see an
```

Comment:
question: What is a span, or is there supposed to be something else here?

Reply:
A "[Span](https://docs.spring.io/spring-cloud-sleuth/docs/current-SNAPSHOT/reference/html/getting-started.html#getting-started-terminology)" is an abstraction in distributed tracing systems that represents a unit of work in a service (in MongoDB a "span" might represent the chunk of time in which 1 Command Event occurred).
 
This is an abstraction outside the context of MongoDB, so I was hesitant to define it, but I can see how it could be jarring to see it here without any context. I've added a link to the `Spring` definition for Span to direct readers to more information if they would like it.